e2e/nomostest/clusterversion/cluster_version.go

```diff
@@ -53,6 +53,29 @@ func (cv ClusterVersion) GKESuffix() (int, error) {
 	return intVal, nil
 }
 
+// IsAtLeast returns whether the ClusterVersion is at least the provided version.
+func (cv ClusterVersion) IsAtLeast(other ClusterVersion) bool {
+	if cv.Major != other.Major {
+		return cv.Major > other.Major
+	}
+	if cv.Minor != other.Minor {
+		return cv.Minor > other.Minor
+	}
+	if cv.Patch != other.Patch {
+		return cv.Patch > other.Patch
+	}
+	// Compare -gke suffix, if they exist
+	if myGKESuffix, err := cv.GKESuffix(); err == nil {
```

Comment:
If one has a gke suffix and then other doesn't, the gke suffix should probably be treated as newer, even if this is unlikely.

Reply:
Updated the behavior